tailwind.config.js

```diff
@@ -56,7 +56,7 @@ module.exports = {
     },
     extend: {
       minWidth: {
-        sidebar: "250px",
+        sidebar: "280px",
```

Comment:
Hero ❤️ 

(I never want to hear that you can't do engineering again)

Reply:
I can't.

https://github.com/railwayapp/docs/pull/546

Reply:
oh lmao

Reply:
Appreciate the hype up though!